### PR TITLE
add K8S 1.16 support

### PR DESCRIPTION
**Pull request by DXist**
This PR adds support of k8s 1.16. Minimum supported k8s version is 1.9.

+ there is a minor fix for polyaxon-streams config - when using SSL for polyaxon-api server and SSL healthchecks healthchecks for streams server use plain HTTP.

### CODE REVIEW

**Comment by mouradmourafiq** [comment]
@DXist Thanks for the contribution, I will review the PR tomorrow or the day after.

File: polyaxon/requirements.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 dependencies:
   - name: postgresql
-    version: 0.8.7
```
Comment (mouradmourafiq) [inline polyaxon/requirements.yaml]: I think this change will have to be made at v1.0, bit for v0.x this will be a huge breaking change for any running cluster.

File: polyaxon/templates/api-deployment.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
```
Comment (mouradmourafiq) [inline polyaxon/templates/api-deployment.yaml]: This makes a lot of sense, it's `apps/v1` for all supported k8s version, i.e.`> v1.12`

File: polyaxon/requirements.yaml (hunk 1 above)
Comment (DXist) [reply polyaxon/requirements.yaml]: reverted dependency on PostgreSQL

File: polyaxon/templates/api-deployment.yaml [hunk 3]
```diff
@@ -113,7 +119,7 @@ spec:
           httpGet:
             path: /healthz/
             port: {{ .Values.streams.service.internalPort }}
-            scheme: {{ .Values.api.service.scheme | default "HTTP" }}
+            scheme: {{ .Values.streams.service.scheme | default "HTTP" }}
```
Comment (mouradmourafiq) [inline polyaxon/templates/api-deployment.yaml]: You can revert this change, the streams service only defines a port, the scheme for all APIs endpoints are set on the api service definition.

Same for the other change in [line 132](https://github.com/polyaxon/polyaxon-chart/pull/51/files#diff-92d5c80b2b2de44841d0bc82b99c7b3cR132)

File: polyaxon/templates/api-deployment.yaml [hunk 4]
```diff
@@ -21,6 +21,10 @@ metadata:
     app.kubernetes.io/managed-by: "helm"
 spec:
   replicas: {{ .Values.api.replicas }}
+  selector:
```
Comment (mouradmourafiq) [inline polyaxon/templates/api-deployment.yaml]: Is the `selector` field required in the 1.16?

File: polyaxon/templates/api-deployment.yaml (hunk 3 above)
Comment (DXist) [reply polyaxon/templates/api-deployment.yaml]: Readiness/liveness probes for streams service don't work when SSL is enabled and the probes for API service are configured to use HTTPS scheme.

Comment (mouradmourafiq) [reply polyaxon/templates/api-deployment.yaml]: I see, it's weird, never had that issue. But it's ok.

File: polyaxon/templates/api-deployment.yaml (hunk 4 above)
Comment (DXist) [reply polyaxon/templates/api-deployment.yaml]: Yes, it's required by apps/v1 spec.

Comment (mouradmourafiq) [reply polyaxon/templates/api-deployment.yaml]: ok thanks for the fix then.

File: polyaxon/templates/api-deployment.yaml (hunk 3 above)
Comment (DXist) [reply polyaxon/templates/api-deployment.yaml]: It's not ok because the chart was deployed with FAILED status.

Does streams service use https when SSL is enabled?
